pages/common/fortune.md

```diff
@@ -0,0 +1,27 @@
+# fortune
+
+> Print a random quotation (fortune-cookie style)
+
+- Print a quotation
+
+`fortune`
+
+- Print a quotation from a given database
+
+`fortune {{database}}`
+
+- Print a list of quotation databases
```

Comment:
It makes sense from this point of view. And they are actually called files in man.

But calling this a database makes sense too.
This is the output of my fortune dbs folder `ls -1 /usr/local/Cellar/fortune/9708/share/`
~~~
art
art.dat
ascii-art
ascii-art.dat
...
~~~

- there are 2 files for each "database"; and when i list them with `fortune -f`, it shows 1 record per database
- files have certain format, so it is not just any text file

Therefore I think just plain file is not very suitable form.

Maybe I should call them dictionary instead of database. What do you think?